Documentation/compression.md

```diff
@@ -0,0 +1,31 @@
+# Compression
+
+## Client side
+The preferred method for configuring message compression on a client is to pass `options` when the client object is instantiated.
+
+The two options need to be passed:
```

Comment:
I suggest something like "These two options control compression behavior". You don't need to provide those options, and you can provide either one without the other.